app/views/content/help-and-advice.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Help and advice
+navigation: 50
+---
+
+We’re here to answer your questions and provide free advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply.
+
+##Come to an event
+
+Our events are a great way to meet people and talk about any concerns or questions you have about becoming a teacher.
+
+###Train to Teach
+
+Train to Teach events are free to attend and you'll be able to:
+
+- put your questions to expert advisers, teachers and training providers
+- chat with current teachers
+- find out what a career in teaching is really like
+- watch presentations which provide detailed guidance on how to get into teaching, the application process and funding your training
+
+
+###Online question and answer sessions
+
+There are online question and answer sessions where you can ask a panel of specialists the questions that matter to you. These sessions cover a range of different topics so you can ask specific questions and receive advice tailored to your particular circumstances.
+
+###School and university events
+
+You could also attend one of the School and University events, either online or in person, and hear directly from teacher training providers about the courses they offer.
+
+Search for an event near you (THIS IS A BUTTON)
+
+##Get a teacher training adviser
+
+Teacher training advisers provide free, one-to-one support wherever you are in your journey into teaching. You can phone, text or email them. They are experienced teachers and able to answer any questions you have. You will have your own dedicated adviser to support you at every step as you make important decisions about your future.
+
+They can help you to:
+
+- get classroom experience
+- find out what teaching is really like
+- understand more about teacher training courses
+- understand the funding and support available to you
+- find the right courses to apply for
+- make a successful application
+- prepare for your interview
+
+Get your adviser (THIS IS A BUTTON)
+
+##Join our mailing list
+
+We can help you get into the classroom with guidance tailored to you, including how to:
+
+- get experience in a classroom
+- find funding to train
+- get one-to-one support
+- read advice from trainees or newly qualified teachers
+- write a successful application
+
+Just answer a few questions about your current situation and what you are interested in and we will keep you up to date with the latest information and advice relevant to you.
```

Comment:
I think this is a bit misleading to talk about 'what you are interested in'? I would probably remove this bit - we are only asking them what subject they want to teach!